webapp/tests/Unit/Controller/API/BaseTest.php

```diff
@@ -84,7 +84,7 @@ protected function removeTestContainer(): void
     }
 
     /**
-     * Verify the given API URL produces the given status code and return the body as JSON
+     * Verify the given API URL produces the given status code and return the body
      *
      * @param string      $method   The HTTP method to use
```

Comment:
I think some of these params can be removed, as they are already in the function signature.

Reply:
But not documented?

Reply:
Ok, I agree.